source/FreeRTOS_IP_Utils.c

```diff
@@ -835,6 +835,51 @@ void prvProcessNetworkDownEvent( struct xNetworkInterface * pxInterface )
     {
         /* The bit 'bEndPointUp' stays low until vIPNetworkUpCalls() is called. */
         pxEndPoint->bits.bEndPointUp = pdFALSE_UNSIGNED;
+
```

Comment:
Since this block of code is exactly same as the code in FreeRTOS_IP.c would it be better to just move this to a function and call it in both the places?
I am saying this because that way it becomes easier to manage just in case we ever need to change anything.
Let me know if that makes sense.

Reply:
I agree. Let me try and clean it a bit and what it looks like with all that repeating code pulled into it's own function.